src/storage/mutate/AddEdgesProcessor.cpp

```diff
@@ -392,8 +392,12 @@ AddEdgesProcessor::findOldValue(PartitionID partId, const folly::StringPiece& ra
                    << ", spaceId " << spaceId_;
         return folly::none;
     }
-    if (iter && iter->valid()) {
-        return iter->val().str();
+
+    while (iter && iter->valid()) {
```

Comment:
No need to modify here. #338 will turn the direction. As for out edge, there won't be any lock.

Reply:
Yeah, 
btw Before merging #338 , please temporarily stop the use of toss